### DISCUSSION TITLE
Enable "tabs on demand" (and/or expose this setting)

### DISCUSSION THREAD

**Discussion by mr-cheffy** [zen-browser/desktop#752] [Ideas]
### What feature would you like to see?

There's a Firefox config flag called `browser.sessionstore.restore_pinned_tabs_on_demand` that lets pinned tabs be restored on demand. I think it would be a good idea to have this exposed in settings and potentially have it enabled by default, so that the browser doesn't use as many resources when restoring from a previous session.

 
 ~ _Created automatically from issue: https://github.com/zen-browser/desktop/issues/709_

**Comment by pret0rian** [comment]
I wouldn't know this if I haven't looked at issue #709 !

Very nice feature and I think it should be easily visible for the end user in some `performance settings`.

Now it is my go-to default ;)
